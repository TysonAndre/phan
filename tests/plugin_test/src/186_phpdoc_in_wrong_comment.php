<?php
/*
 * @param string $value
 */
function my_strlen($value) {
    return strlen($value);
}
echo my_strlen('test');
