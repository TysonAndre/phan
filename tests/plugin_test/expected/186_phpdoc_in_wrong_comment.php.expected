src/186_phpdoc_in_wrong_comment.php:2 PhanPluginPHPDocInWrongComment Saw possible phpdoc annotation in ordinary block comment "/*\n * @param string $value\n */". PHPDoc comments should start with "/**", not "/*"
